.doc_gen/metadata/sqs_metadata.yaml

```diff
@@ -332,9 +332,9 @@ sqs_SendMessage:
         - sdk_version: 1
           github: rust_dev_preview/sqs
           excerpts:
-            - description:
+            - description: Send message to &SQS; queue.

```

Comment:
The synopsis for the example (at line 288) renders on the same page as the example tablist, so include a description here only if you're explaining something special or different about the Rust example itself.

Reply:
Done.